src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/WindowsRuntime/Generated/WinRT/ComWrappersSupport.net5.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using WinRT.Interop;
+using static System.Runtime.InteropServices.ComWrappers;
+
+namespace WinRT
+{
+    internal static partial class ComWrappersSupport
+    {
+        public static readonly ConditionalWeakTable<object, InspectableInfo> InspectableInfoTable = new ConditionalWeakTable<object, InspectableInfo>();
+
+        private static ComWrappers ComWrappers { get; } = new WpfWinRTComWrappers();
+
+        public static unsafe InspectableInfo GetInspectableInfo(IntPtr pThis)
+        {
+            var _this = FindObject<object>(pThis);
```

Comment:
Maybe we can use `@this` 

```suggestion
            var @this = FindObject<object>(pThis);
```

Reply:
As mentioned previously, I'd rather avoid making style changes to the generated or imported code unless absolutely required. Keeping it the same when possible makes it easier to maintain.

Reply:
@lindexi This is all generated code and intended to be removed as soon as the CsWinRT tool can be consumed. I agree with @jkoritzinsky that updating this with stylistic changes doesn't help actually address the reason for this change.

Reply:
Since this is in master and not p6, why not introduce cswinrt now instead of adding quasi-generated sources? Is the tool not ready?

Reply:
The correct manner is for the SDK to be shipped as part of the `net5.0-windows` tfm. We aren't quite at the point for that change yet, so for now we're doing this (embedding the sources) for master and p6 if this gets cherry-picked.